internal/testutil/testutil.go

```diff
@@ -227,9 +230,17 @@ func (m *Manager) GetCurrentResourceSlices(ctx context.Context) ([]*apiv1.Resour
 	return returns, nil
 }
 
+var Backoff = wait.Backoff{
+	Steps:    10,
+	Duration: 10 * time.Millisecond,
+	Factor:   2.0,
+	Jitter:   0.1,
+	Cap:      time.Minute,
+}
+
 func Eventually(t testing.TB, fn func() bool) {
 	t.Helper()
-	SomewhatEventually(t, time.Second*5, fn)
+	SomewhatEventually(t, time.Second*15, fn)
```

Comment:
This probably isn't necessary now that the fixes have been found, but it helps get signal on future races